bika/lims/upgrade/v01_03_001.py

```diff
@@ -42,7 +44,60 @@ def upgrade(tool):
     logger.info("Upgrading {0}: {1} -> {2}".format(product, ver_from, version))
 
     # -------- ADD YOUR STUFF BELOW --------
+    setup.runImportStepFromProfile(profile, 'actions')
 
+    # https://github.com/senaite/senaite.core/pull/1324
+    # initialize the auditlog
+    init_auditlog(portal)
 
     logger.info("{0} upgraded to version {1}".format(product, version))
     return True
+
+
+def init_auditlog(portal):
+    """Initialize the contents for the audit log
+    """
+    logger.info("Initialize contents for audit logging")
+
+    from bika.lims.subscribers.auditlog import has_snapshots
+    from bika.lims.subscribers.auditlog import take_snapshot
+    from bika.lims.subscribers.auditlog import is_auditable
+    from bika.lims.api.security import get_user
+    from bika.lims.api.security import get_roles
+    from DateTime import DateTime
+    from bika.lims.interfaces import IAnalysis
```

Comment:
Can you move this imports at the top or is there any reason why you have to add them here?

Reply:
Same as above. I think the `has_snapshots`, `take_snapshot`, `is_auditable` should live somewhere else...